### ISSUE TITLE
[iOS] KMEI keyboard initialization with non-default keyboard

### ISSUE THREAD

**Issue by jahorton** [keymanapp/keyman#2160]
**Describe the bug**
Noted during work on https://github.com/keymanapp/help.keyman.com/pull/58 - KMEI currently lacks a well-documented way for engine users to easily establish the first keyboard to load.

KMEA allows app developers to set an initial keyboard for use on app launch.  Attempting similar code in KMEI results in a broken keyboard state.  This occurs when `setKeyboard` is called before the `keyboardLoaded` function of `KeymanWebViewController` is called; the former will set `shouldReloadKeyboard = false` while being unable to actually load the keyboard (as KMW isn't ready), causing the second one - which triggers default keyboard loading (when `setKeyboard` isn't called) to not load the keyboard.

**To Reproduce**

1. Please refer to https://help.keyman.com/DEVELOPER/engine/iphone-and-ipad/12.0/guides/in-app/ (once the above-mentioned PR is merged).
2. At the end of the guide's "Step 4", insert a `setKeyboard` call, in context:

```
Manager.shared.addKeyboard(kb);

// The important line!!!
Manager.shared.setKeyboard(kb);
// End "important line"
// ...
```

Feel free to use either `kb` or `Defaults.keyboard` - both will fail!

**Expected behavior**
The specified keyboard would always be the initial keyboard displayed within the KMEI-consuming app.

This would likely be best accomplished through the following changes:

1. `InputViewController` / `KeymanWebViewController` should expose a property (internally) that reflects the current 'keyboard loaded?' state.
2. If the keyboard isn't loaded, `shouldReloadKeyboard` will not be modified by `Manager.setKeyboard`.  This should allow the `keyboardLoaded` event to retrieve the new setting and performed a 'deferred' load automatically with minimal code changes.

**Screenshots**

With a `setKeyboard()` call:

![kmei-set-keyboard-bug](https://user-images.githubusercontent.com/25213402/66030526-c0a67680-e52b-11e9-9276-6293f86ed615.png)

Without:

![kmei-default-kbd-load](https://user-images.githubusercontent.com/25213402/66030801-58a46000-e52c-11e9-990b-83ed1ab238f7.png)

(Note that the keyboard shown was the last keyboard set on a previous load.)

---
  
**iPhone/iPad:**
 - Device: testable on Simulator
 - OS: 12.2
 - Keyman version 12.0.44
 - Target application: consumers of KMEI (KMSample1 being a great example case)

**Comment by jahorton** [comment]
I'm fairly certain this has already been fixed by #4771 and #4808; I forgot this issue existed at the time.
